### ISSUE TITLE
Investigate better video conferencing solutions

### ISSUE THREAD

**Issue by JKrag** [Praqma/praqma-rnd#13]
I feel we are pushing the limits of Google Hangouts these days, especially for bigger meetings. The amount of time we spend fixing lost audio and frequent disconnections is simply to big and too annoying.

Maybe it is time to slowly look at alternatives?

This could be better software solutions, or maybe we are even at a point where we could benefit from some _dedicated hardware_ solution, at least between the main meeting rooms at each site?

There are solutions out there that actually work - the question is whether they are payable and would work for us. E.g. accessing from a laptop with pure software from somewhere else is an absolute needed requirement.

Cisco's hardware solution might be way over our needs and budget, but there might be intermediate level dedicated compromises available, or maybe just software solutions that are more stable.

Feel free to comment on this if you have any suggestions worth looking into - maybe stuff you have worked with previously or at customers.

Then, when somebody has time, they can look more closely and compare.

**Comment by Andrey9kin** [comment]
I agree. Let's start from building a list of available options that we find decent. https://zoom.us/ seems to be widely used

**Comment by buep** [comment]
> Feel free to comment on this if you have any suggestions worth looking into - maybe stuff you have worked with previously or at customers.

I never - yet - used a technology at a customer site that just worked. Always trouble and always problems with it doesn't work in all browsers and especially on Linux.
